server/app/views/applicant/ApplicantLayout.java

```diff
@@ -95,11 +95,12 @@ private Content renderWithSupportFooter(HtmlBundle bundle, Messages messages) {
 
     DivTag supportLink =
         div()
+            .withClasses("flex", "flex-row")
             .with(
-                text(messages.at(MessageKey.FOOTER_SUPPORT_LINK_DESCRIPTION.getKeyName())),
-                text(" "),
-                emailAction)
-            .withClasses("mx-auto", "max-w-screen-sm", "w-5/6");
+                div()
+                    .withText(messages.at(MessageKey.FOOTER_SUPPORT_LINK_DESCRIPTION.getKeyName()))
+                    .with(text(" "), emailAction)
```

Comment:
nit: if it doesn't affect the UI, i think the old way of having everything in one with() is a bit cleaner.

Reply:
It does affect the UI, the div was needed to get this centered afaict.

Reply:
I meant keeping the inner div, but going from
```
div().withText(<message>).with(text(" "), emailAction)
```
to 
```
div().with(text(<message>), text(" "), emailAction)
```

Reply:
Gotcha, that worked 👍